app/Database/Requirement.hs

```diff
@@ -0,0 +1,59 @@
+{-|
+Description: Requirement
+
+Module containing data type that represents a "Requirement".
+
+We will use parsed data to create instances of this type.
+-}
+module Requirement
+( ProgramReq(..)
+, CourseRequirement(..)
+, CourseReq(..)
+, Req(..)
+, showProgramReq
+, showCourseRequirement
+, showCourseReq
+, showReq
+) where
+
+import Data.String
+
+
+data ProgramReq = PRGREQ String [Req]
+
+-- | Returns a well formatted String representing a program requirement for specified program.
+showProgramReq :: ProgramReq -> String
+showProgramReq (PRGREQ program reqs) = "The program requirements for " + program + ":\n"
+										+ map (showReq) reqs
+
+
+-- Assuming we have correctly parsed inputs, CourseReq is in order [corequisite, exclusion, prerequisite]
```

Comment:
This isn't a good comment - you shouldn't assume anything about the order of the requirements in the list `[CourseReq]`.